src/AccessibilityInsights/MainWindowHelpers/AutoUpdate.cs

```diff
@@ -143,13 +150,14 @@ private async void DownLoadInstaller(IAutoUpdate autoUpdate, AutoUpdateOption up
                 e.ReportException();
             };
 
+            Topmost = oldTopMost;
             ctrlProgressRing.Deactivate();
             Logger.PublishTelemetryEvent(TelemetryAction.Upgrade_InstallationError, TelemetryProperty.Error, result.ToString());
 
             string message = updateOption == AutoUpdateOption.RequiredUpgrade
                 ? GetMessageForRequiredUpdateFailure() : GetMessageForOptionalUpdateFailure();
 
-            MessageBox.Show(message);
+            MessageDialog.Show(message);
```

Comment:
Curious: Why the switch to MessageDialog? I'm guessing MessageDialog is a common component? If so, should we use it everywhere in our UI?

Reply:
Message dialog is our custom styled (and hopefully more accessible) version of MessageBox. Yes, I believe we should use it everywhere, unless there is a specific reason to use an alternative.